packages/neon-core/__tests__/tx/transaction/EnrollmentTransaction.ts

```diff
@@ -0,0 +1,137 @@
+import { Transaction, TransactionLike, TransactionType } from "../../../src/tx";
+import EnrollmentTransaction, {
+  EnrollmentTransactionLike
+} from "../../../src/tx/transaction/EnrollmentTransaction";
+
+import { BaseTransaction } from "../../../src/tx/transaction/BaseTransaction";
+import samples from "./enrollmentTx.json";
+
+describe("constructor", () => {
+  test("empty", () => {
+    const result = new EnrollmentTransaction();
+
+    expect(result).toBeInstanceOf(EnrollmentTransaction);
+    expect(result).toBeInstanceOf(BaseTransaction);
+    expect(result.type).toBe(TransactionType.EnrollmentTransaction);
+  });
+
+  test("TransactionLike", () => {
+    const testObject = {
+      version: 1,

```

Comment:
lets stick with a v0 transaction. Minor but don't want the tests using supposedly illegal values.

Reply:
hi you have not addressed this yet.

Reply:
Sorry, I missed that. :sweat_smile: